### DISCUSSION TITLE
Road Map

### DISCUSSION THREAD

**Discussion by TSmiley12** [horilla-opensource/horilla#326] [Q&A]
Am Thomas and I have been happy with progress of this open source app. Am only asking when the other apps in the road map will be released.

**Comment by horilla-opensource** [comment]
> Am Thomas and I have been happy with progress of this open source app. Am only asking when the other apps in the road map will be released.

Hi @TSmiley12 ,

Thank you for your feedback!

We're excited to share that our upcoming version includes several new features currently under development:

- **Project Management**
- **Calendar Integration**
- **Real-Time Chat**

In addition, we're making significant improvements to the codebase. We're transitioning to a more generic architecture that will empower developers to easily customize and extend the platform to meet specific needs. This change is aimed at providing greater flexibility and ease of use for the developer community.

We appreciate your continued support and look forward to delivering these updates in the next release.

With regards,
**Team Horilla**

**Reply by surenyfs** [reply]
We are waiting for the project management. Is there any approximate timeline for the same?.

**Reply by horilla-opensource** [reply]
> We are waiting for the project management. Is there any approximate timeline for the same?.

We are currently focusing on closing on our version 1 with some minor fixes. I will update with you once we fix the final schedule.

**Reply by hendricjabs** [reply]
Do you have any news regarding the project management feature?

**Reply by horilla-opensource** [reply]
Hi @hendricjabs ,

The project module was already published to the master code base. But the sidebar was missing it's menu. It's now updated.

You can pull the latest code from the master branch and check again.

With Regards,
**Team Horilla**
